eth/tracers/native/call.go

```diff
@@ -264,13 +263,15 @@ func (t *callTracer) GetResult() (json.RawMessage, error) {
 
 	if t.callstack[0].Type == vm.STOP {
 		t.callstack[0].Error = "failed deposit transaction"
+		t.callstack[0].To = nil
+		t.callstack[0].Gas = 0
```

Comment:
Modification from upstream to ensure consistency in deposit tracing. Should be covered by the tests added in https://github.com/ethereum-optimism/op-geth/pull/333

(note there are similar changes in call_flat.go too)

Reply:
Nice to so the tests cover the nil/0 part here